.docker/Dockerfile.dev-release

```
@@ -1,5 +1,5 @@
 FROM docker.io/debian:stable-slim
-WORKDIR /mm2
-COPY target/release/mm2 /usr/local/bin/mm2
+WORKDIR /kdf
+COPY target/release/kdf /usr/local/bin/kdf
```

Let's ship `mm2` binary into containers as well:

```suggestion
COPY target/release/kdf /usr/local/bin/kdf
COPY target/release/mm2 /usr/local/bin/mm2
```